client/components/domains/register-domain-step/index.jsx

```diff
@@ -1790,6 +1799,15 @@ class RegisterDomainStep extends Component {
 		} );
 	}
 
+	clearSuggestionErrorMessage() {
+		this.setState( {
+			showSuggestionNotice: false,
+			suggestionError: null,
+			suggestionErrorData: null,
+			suggestionErrorDomain: null,
```

Comment:
```suggestion
			showSuggestionNotice: false,
			suggestionError: null,
			suggestionErrorData: null,
			suggestionErrorDomain: null,
			showAvailabilityNotice: false,
			availabilityError: null,
			availabilityErrorData: null,
			availabilityErrorDomain: null,
```
What do you think of also clearing the availability error here too? Otherwise, if we go to the domain-only flow `/start/domain/domain-only?search=yes&new=test202411213.linkasdf` (domain that doesn't exist) the `test202411213.linkasdf isn't a valid domain` error notice will appear and will still be there if we go to the Gravatar flow `/start/domain-for-gravatar/domain-only?search=yes&new=test202411213.link`.